language/stdlib/modules/LibraTransactionTimeout.move

```diff
@@ -25,7 +25,7 @@ module LibraTransactionTimeout {
 
   public fun is_valid_transaction_timestamp(timestamp: u64): bool acquires TTL {
     // Reject timestamp greater than u64::MAX / 1_000_000;
-    if(timestamp > 9223372036854) {
+    if(timestamp > 18446744073709) {
```

Comment:
@runtian-zhou I wound up changing this since it was set to `std::i64::MAX/1_000_000` instead of `std::u64::MAX`, let me know if this change looks OK.